src/lib.rs

```diff
@@ -213,7 +213,6 @@ mod test {
     #[test]
     fn option() {
         assert_implicit_clone!(Some("foo"));
-        // `assert_implicit_clone!(Some(NonImplicitCloneType));` doesn't compile
```

Comment:
`-fail` test would cover this lol

Reply:
what do you mean? I don't understand

Reply:
if there were -fail tests like for macros, then this "doesn't compile" example would be put there